sdk/ml/azure-ai-ml/azure/ai/ml/operations/_job_operations.py

```diff
@@ -384,9 +443,8 @@ def _try_get_compute_arm_id(self, compute: Union[Compute, str]):
                 compute_name = str(compute)
             else:
                 raise ValueError(
-                    "compute must be either an arm id of Compute, a Compute object or a compute name but got {}".format(
-                        type(compute)
-                    )
+                    "compute must be either an arm id of Compute, a Compute object or a compute name but"
```

Comment:
Sanity check - is this valid syntax? It looks like there ought to be a "+" or some other symbol binding the two strings together.

Reply:
Yep, it's valid. Here's what happens when it's triggered:

![image](https://github.com/Azure/azure-sdk-for-python/assets/16376603/8c1c35f8-6424-4914-bd08-2e170c7884df)